bepolia/config.yaml

```diff
@@ -29,8 +29,8 @@ TERMINAL_BLOCK_HASH: 0x000000000000000000000000000000000000000000000000000000000
 TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
 
 # Capella
-CAPELLA_FORK_VERSION: 0x03001020
-CAPELLA_FORK_EPOCH: 18446744073709551615
+CAPELLA_FORK_VERSION: 0x90000072
+CAPELLA_FORK_EPOCH: 56832
 
 # Sharding
 SHARDING_FORK_VERSION: 0x04001020
```

Comment:
We should also probably rename these to be:
```
# Deneb
DENEB_FORK_VERSION: 0x04001020
DENEB_FORK_EPOCH: 18446744073709551615
```
So that it conforms with the latest mainnet config naming. I'm not sure if all clients already support/recognize the new field though.

Reply:
Done, thanks @parithosh !